libs/core/toolbar/toolbar.component.ts

```diff
@@ -168,6 +168,9 @@ export class ToolbarComponent implements AfterViewInit, AfterViewChecked, CssCla
         return this._titleComponent$.value;
     }
 
+    /** @hidden */
+    @ContentChildren(ToolbarSpacerDirective) spacer: QueryList<ToolbarSpacerDirective>;
```

Comment:
you can use signals for this as well: https://angular.dev/guide/signals/queries#contentchildren
spacers = contentChildren(ToolbarSpacerDirective);

Reply:
check out the changes I have pushed